src/drafts/MarkdownEditor/_FormattingTools.tsx

```diff
@@ -49,6 +48,8 @@ export const FormattingTools = forwardRef<FormattingTools, {forInputId: string}>
     reference: () => referenceRef.current?.click()
   }))
 
+  require('@github/markdown-toolbar-element')
```

Comment:
I don't think doing this in component scope is ideal, sine this calls require on every render (sure it's often a noop), but also runs in server environments. 

```
useEffect(() => { require....}, [])
```

would at least ensure it doesn't run in non-browser environments, however I believe that was already tried and lead to the issues with `require'ing` an esmodule.

Reply:
evaluating this require in a context that runs on the server likely explains the CI error @joshblack  noted 

```
error Building static HTML failed

  34 | }
  35 | const styles = new WeakMap();
> 36 | class MarkdownButtonElement extends HTMLElement {
     | ^
  37 |     constructor() {
  38 |         super();
  39 |         const apply = () => {


  WebpackError:ReferenceError: HTMLElement is not defined
  ```

Reply:
Yup! Me and @joshblack paired on this earlier and I just forgot to push the commit.